blog/2023-06-28-vue-tutorial.md

```diff
@@ -272,27 +316,68 @@ import { T } from '@tolgee/vue'
   <div class="greetings">
    ...
     <h3>
-      <T keyName="success" defaultValue="You’ve successfully created a project with Vite + Vue 3" />
+      <T keyName="success" defaultValue="You've successfully created a project with Vite + Vue 3" />
     </h3>
   </div>
 </template>
 ```
 
-Finally, update your `App.vue` file with the code below:
+Finally, update your App.vue file with the code below:
 
-In the code block above, we have updated our components imports and added the `Selector` component we created earlier.
+```typescript
+<script>
+import { TolgeeProvider, T } from '@tolgee/vue'
+import HelloWorld from './components/HelloWorld.vue'
+import TheWelcome from './components/TheWelcome.vue'
+import { defineComponent } from 'vue'
+export default defineComponent({
+  components: { HelloWorld, TheWelcome, TolgeeProvider, T },
+  setup() {}
+})
+</script>
 
-But there is one last thing we need to do, which is to add our translation files to the Vue project.
+<template>
+  <TolgeeProvider>
+    <header>
+      <img alt="Vue logo" class="logo" src="./assets/logo.svg" width="125" height="125" />
 
-In your Tolgee dashboard, click on export in the left menu, then select the languages you wish to use and click on export.
+      <div class="wrapper">
+        <HelloWorld :msg="$t('hello')" />
+      </div>
+    </header>
+    <main>
+      <TheWelcome />
+    </main>
+  </TolgeeProvider>
+</template>
+```
 
-![Export translations from Tolgee](/img/blog/vue-tutorial/export_translations.png)
+In the code block above, we have updated our components imports and added the Selector component we created earlier.
+
+## Step 9: Translation
+
+At this stage, to translate a text on your website, press and hold the `alt` key or `option` key and hover the target text. The text will be highlighted like the screenshot below:
 
-Create i18n folder in your Vue application public folder and add the extracted files
+![Text Highlighted](/img/blog/vue-tutorial/alt_click.png)
 
-![Extracted files](/img/blog/vue-tutorial/extracted_files.png)
+Then click on the highlighted zone to open the translation dialog.
+
+![Translation dialog](/img/blog/vue-tutorial/translation_dialog.png)
+
+Once you have added your translations, click on the save button to store your translations in the Tolgee Platform.
+
+Equally, you can follow the same process to update your translations.
+Repeat this process for every string you want to translate and with a simple click Tolgee will save your translations. If you open your project in the platform, you should have a list of translated strings reflected in the platform
+
+![List of translations](/img/blog/vue-tutorial/translation_list.png)
+
+Tolgee also allows you to export your translations in `json` files. In your Tolgee dashboard, click on export in the left menu, then select the languages you want then click on export
```

Comment:
You should mention here, that this can be done automatically in CI/CD tool via REST API or Tolgee CLI. So user's wont think that they have to manually export it all the time.